phalcon/db/dialect/sqlite.zep

```diff
@@ -90,7 +90,11 @@ class Sqlite extends Dialect
 					let columnSql .= "DATETIME";
 				}
 				break;
-
+			case Column::TYPE_TIME:
+				if empty columnSql {
+					let columnSql .= "TIME";
```

Comment:
Are you sure about this? https://www.sqlite.org/datatype3.html#date_and_time_datatype

That's why I asked you to provide a workaround. That is why we are not in a hurry to take all possible data types.